docs/dev/how-to/debug_permissions.mdx

```diff
@@ -0,0 +1,68 @@
+# Debugging permissions
+
+This document provides a list of options to try when debugging permissions issues.
+
+## Check the user permissions screen
+
+1. As a site administrator, visit the **Site admin** > **User administration** page and search for the user in question.
+![](https://storage.googleapis.com/sourcegraph-assets/Docs/how-to/debug-permissions/user_search.png)
+2. Click on their username and navigate to their **Settings** > **Permissions** page.
+   This page gives an overview of a user's permission sync jobs, as well as the repositories they have access to and why.
+![](https://storage.googleapis.com/sourcegraph-assets/Docs/how-to/debug-permissions/permissions_page.png)
+3. At the bottom of the **Permissions** page you'll find a list of repositories the user has access to and the reason for their access. Possible access reasons are:
+    - **Public** - The repository is marked as public on the code host, so all users on the Sourcegraph instance can access this repository.
+    - **Unrestricted** - The code host connection that this repository belongs to is not enforcing authorization, so all users on the Sourcegraph instance can access this repository.
+    - **Permissions Sync** - The user has access to this repository because they have connected an external account that has access to this repository on the code host.
```

Comment:
```suggestion
    - **Permissions Sync** - The user has access to this repository because they have connected an external account with access to it on the code host.
```